pinot-common/src/main/java/com/linkedin/pinot/startree/hll/HllConfig.java

```diff
@@ -30,8 +31,11 @@
  * segment builder will generate corresponding hll derived fields on the fly.
  */
 public class HllConfig {
+  private static final ImmutableBiMap<Integer, Integer> LOG2M_TO_SIZE_IN_BYTES =
+      ImmutableBiMap.of(5, 32, 6, 52, 7, 96, 8, 180, 9, 352);
+
   private int hllLog2m = HllConstants.DEFAULT_LOG2M;
-  private int hllFieldSize = HllUtil.getHllFieldSizeFromLog2m(HllConstants.DEFAULT_LOG2M);
+  private int hllFieldSize = getHllFieldSizeFromLog2m(HllConstants.DEFAULT_LOG2M);
```

Comment:
is LOG2M_TO_SIZE_IN_BYTES duplicated ? Why not continue to use HllUtil ?

Reply:
Originally, I didn't want to use HllUtil mainly because it has a method where it pulls in GenericRow, currently in pinot-core also. I've changed it so that I pull out the logic having to do with hllfieldsize into a different HllSizeUtils and put that in pinot-common so that no logic is duplicated.